react/components/CollaboratorsList/index.js

```diff
@@ -0,0 +1,54 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import { graphql } from 'react-apollo';
+import styled from 'styled-components';
+
+import collaboratorsListQuery from 'react/components/CollaboratorsList/queries/collaboratorsList';
+
+import Modal from 'react/components/UI/Modal';
+import ManageCollaborators from 'react/components/ManageCollaborators';
+import CollaboratorsList from 'react/components/CollaboratorsList/CollaboratorsList';
+
+const StyledCollaboratorsList = styled(CollaboratorsList)`
+  margin-bottom: 1em;
+`;
+
+class CollaboratorsListContainer extends Component {
+  static propTypes = {
+    channel_id: PropTypes.number.isRequired,
+    // HACK: Until we wire up SSR; this requires an HTML fragment
+    // to render while loading.
+    htmlFragment: PropTypes.string.isRequired,
```

Comment:
This is probably the only mildly controversial thing in this PR. I haven't wired up SSR yet so instead this just takes over the HTML fragment while it's loading so we don't get a blank flash.